spec/system/viewing_reviewable_spec.rb

```diff
@@ -108,13 +109,87 @@
       end
 
       it "allows to add notes and persists them when toggle tabs" do
-        review_page.visit_reviewable(reviewable_flagged_post)
-        review_page.click_timeline_tab
+        refreshed_review_page.visit_reviewable(reviewable_flagged_post)
+        refreshed_review_page.click_timeline_tab
         review_note_form.add_note("This is a review note.")
-        review_page.click_insights_tab
-        review_page.click_timeline_tab
+        refreshed_review_page.click_insights_tab
+        refreshed_review_page.click_timeline_tab
         expect(page).to have_text("This is a review note.")
       end
     end
+
+    describe "when the reviewable item is a user" do
+      fab!(:user)
+      let(:rejection_reason_modal) { PageObjects::Modals::RejectReasonReviewable.new }
+      let(:scrub_user_modal) { PageObjects::Modals::ScrubRejectedUser.new }
+
+      before do
+        SiteSetting.must_approve_users = true
+        Jobs.run_immediately!
+        user.update!(approved: false)
+        user.activate
+      end
+
+      it "shows the user's name and admin profile link" do
+        reviewable = ReviewableUser.find_by_target_id(user.id)
+
+        refreshed_review_page.visit_reviewable(reviewable)
+        expect(page).to have_text(user.name)
+        expect(page).to have_link(user.username, href: "/admin/users/#{user.id}/#{user.username}")
+      end
+
+      it "Allow to delete user" do
+        reviewable = ReviewableUser.find_by_target_id(user.id)
+        user_email = user.email
+
+        refreshed_review_page.visit_reviewable(reviewable)
+        refreshed_review_page.select_bundled_action(reviewable, "user-delete_user")
+        expect(refreshed_review_page).to have_reviewable_with_rejected_status(reviewable)
+
+        mail = ActionMailer::Base.deliveries.first
+        expect(mail.to).to eq([user_email])
+        expect(mail.subject).to match(/You've been rejected on Discourse/)
+      end
+
+      it "Allows scrubbing user data after rejection" do
+        scrubbing_reason = "a spammer who knows how to make GDPR requests"
+        reviewable = ReviewableUser.find_by_target_id(user.id)
+        user_email = user.email
+
+        refreshed_review_page.visit_reviewable(reviewable)
+        refreshed_review_page.select_bundled_action(reviewable, "user-delete_user")
+
+        expect(refreshed_review_page).to have_scrub_button(reviewable)
+        refreshed_review_page.click_scrub_button(reviewable)
+
+        expect(scrub_user_modal.scrub_button).to be_disabled
+        scrub_user_modal.fill_in_scrub_reason(scrubbing_reason)
+        expect(scrub_user_modal.scrub_button).not_to be_disabled
+        scrub_user_modal.scrub_button.click
```

Comment:
After scrubbing, the review page becomes unusable if you visit the page because of an error.

```
insights.gjs:34 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'flags_agreed')
    at get reviewInsights (insights.gjs:34:14)
```